apps/vs-code-designer/src/app/utils/azurite/activateAzurite.ts

```diff
@@ -53,7 +51,7 @@ export async function activateAzurite(context: IActionContext): Promise<void> {
 
       // User has not configured workspace azurite.location.
       if (!azuriteLocationExtSetting) {
-        const defaultAzuriteDir = path.join(os.homedir(), '.logicapps', 'azurite');
+        const defaultAzuriteDir = defaultAzuritePathValue;
         const azuriteDir = await context.ui.showInputBox({
           placeHolder: localize('configureAzuriteLocation', 'Azurite Location'),
           prompt: localize('configureWebhookEndpointPrompt', 'Configure Azurite Workspace location folder path'),
```

Comment:
"workspace" isn't capitalized: https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=visual-studio

```suggestion
          prompt: localize('configureWebhookEndpointPrompt', 'Configure Azurite workspace location folder path'),
```